src/dashboard/src/components/administration/tests/test_dip_upload_configs.py

```diff
@@ -1,28 +1,24 @@
 import os
 
 from django.core.urlresolvers import reverse
-from django.http import Http404, HttpResponse
 from django.test import TestCase
-from django.test.client import Client
 
 from components import helpers
 from components.administration.views_dip_upload import _AS_DICTNAME, _ATOM_DICTNAME
 from main.models import DashboardSetting
 
-import mock
-
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
-FIXTURES_DIR = os.path.abspath(os.path.join(THIS_DIR, "../../../tests/fixtures"))
+FIXTURES_DIR = os.path.abspath(os.path.join(THIS_DIR, "../../../../tests/fixtures"))
```

Comment:
Hi @jraddaoui this has been approved so feel free to ignore this suggested change, but should this `FIXTURES_DIR` path be set using something a little more idiomatic/platform independent? (Same for the processing configuration test  below)
```
FIXTURES_DIR = os.path.join(
    os.path.dirname(
        os.path.dirname(
            os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
        )
    ),
    "tests",
    "fixtures",
)
```
**NB.** This is just one way known to me, there may be a better approach.

Though, the complexity of this leads me back to your original question about where the tests should sit. In line with the tests belonging to their own applications, should the fixtures for each of the components be independent of each other too? In that case we'd create a fixtures dir for each of the components?

Reply:
Good point @ross-spencer!

Not a big deal, but that made me realize that we were building the fixture paths in all tests, when that can be done directly in the Django settings. Please, take a look when you can to the last commit ;)

Reply:
That's a nice improvement Radda.